### ISSUE TITLE
Have a single point of entry for the application.

### ISSUE THREAD

**Issue by glen** [rocky02/q_it#5]
Currently - the application has `app.rb` which is the starting point for publisher.
The subscriber (in the feature/consume_messages_from_aws_queue branch) - is executed by calling another file `subscriber.rb`.
Have a single point of entry to the application with both the publisher and the subscriber being able to be called from `app.rb`.
Ensure that you can differentiate the two services by a standard keyword - for example 
`ruby app.rb [publish|subscribe] [queue_name] [publish|check message frequency]`

Validations should happen prior to objects being created or at the time of object creation - either is fine - but not after creating the object or at the time of publishing/subscribing.

Acceptance Criteria
1. Ability to run the publisher through app.rb
2. Ability to run the subscriber through app.rb
3. Have multiple terminals open and multiple publishers run and subscribers run concurrently.

**Comment by rocky02** [comment]
What kind of validations are we talking about? Currently we are handling validations for sleep_time but that is after object creation.

**Comment by glen** [comment]
The same validation - though we should do it prior to object creation.

**Comment by rocky02** [comment]
Working on issue in PR #6

**Comment by rocky02** [comment]
> The same validation - though we should do it prior to object creation.

will handle validations in separate PR.

**Comment by glen** [comment]
Merged to master branch
